javascript-modal/main.js

```diff
@@ -0,0 +1,24 @@
+var isOff = false;
+
+var $openModal = document.querySelector('.open-button');
+var $modalContainer = document.querySelector('.modal-container');
+
+$openModal.addEventListener('click', function popup(event) {
+  isOff = !isOff;
```

Comment:
Since you have this functionality in multiple different functions, a boolean flag is not needed. As it is right now, for both of these functions, you only update the DOM if the variable value is true, so you are flipping the value, checking the value, and flipping it right back. Because of this, when your function runs, the value will always start as `false`, get flipped to `true`, and then back to `false`. AKA, every time the function runs, the same series of events take place with no variance. You can make use of a boolean flag if you only made one event handler function, but with two separate functions, you can just do the work without needing to check anything. If somebody clicks the "Open Modal" button, they want to open the modal every time. If somebody clicks the "No" button, they want to close the modal every time.